### ISSUE TITLE
Clicking result line no longer showing details

### ISSUE THREAD

**Issue by Keithkdl** [wallabyjs/public#2219]
### Issue description or question

I think there was an update yesterday, 24th Sept, which made the result screen (in browser) stop showing the details of a test when you click on it.  I think this as i'm sure i saw wallaby being updated, then the problem started right after.

I have uninstalled wallaby from vscode, restarted, then installed again, and the problem persists.  Using http://localhost:51245/ or https://wallabyjs.com/app/#/tests has the same problem

### Wallaby diagnostics report

```
{ editorVersion: '1.38.1',
  pluginVersion: '1.0.143',
  editorType: 'VSCode',
  osVersion: 'win32 10.0.18362',
  nodeVersion: 'v10.16.3',
  coreVersion: '1.0.752',
  config:
   { files:
      [ { pattern: 'src/**/*.ts', load: false, ignore: false, trigger: true, instrument: true, order: 1 },
        { pattern: 'src/**/*.d.ts', ignore: true, trigger: true, load: true, instrument: true },
        { pattern: 'src/**/student*spec.ts', ignore: true, trigger: true, load: true, instrument: true } ],
     filesWithNoCoverageCalculated: [ 'src/server.ts', 'src/routes.ts', 'src/index.ts', 'src/database.ts', 'src/controllers/setup.controller.ts' ],
     tests: [ { pattern: 'src/**/*.spec.ts', load: true, ignore: false, trigger: true, test: true, order: 2 } ],
     env: { type: 'node', params: {}, runner: 'C:\\Program Files\\nodejs\\node.exe', viewportSize: { width: 800, height: 600 }, options: { width: 800, height: 600 }, bundle: true },
     testFramework: { version: 'jasmine@2.3.4', configurator: 'jasmine@2.3.4', reporter: 'jasmine@2.3.4', starter: 'jasmine@2.3.4' },
     compilers: { '**/*.?(lit)coffee?(.md)': [Function] },
     debug: false,
     name: 'Grading Companion NodeJS',
     slowTestThreshold: 200,
     lowCoverageThreshold: 90,
     maxConsoleMessagesPerTest: 100,
     reportConsoleErrorAsError: true,
     workers: { restart: true, initial: 0, regular: 0, recycle: true },
     runAllTestsInAffectedTestFile: false,
     preprocessors: {},
     autoConsoleLog: true,
     delays: { run: 0, edit: 100, update: 0 },
     teardown: undefined,
     hints:
      { ignoreCoverage: '__REGEXP /ignore coverage|istanbul ignore/',
        ignoreCoverageForFile: '__REGEXP /ignore file coverage/',
        commentAutoLog: '?',
        testFileSelection: { include: '__REGEXP /file\\.only/', exclude: '__REGEXP /file\\.skip/' } },
     automaticTestFileSelection: true,
     runSelectedTestsOnly: false,
     extensions: {},
     reportUnhandledPromises: true,
     loose: undefined,
     configCode:
      'var compilerOptions = Object.assign(require(\'./tsconfig.json\').compilerOptions);\r\n\r\nmodule.exports = function ( wallaby ) {\r\n    return {\r\n        files: [\r\n            { pattern: \'src/**/*.ts\', load: false },\r\n            { pattern: \'src/**/*.d.ts\', ignore: true },\r\n            { pattern: \'src/**/student*spec.ts\', ignore: true }\r\n        ],\r\n        filesWithNoCoverageCalculated: [\r\n            \'src/server.ts\',\r\n            \'src/routes.ts\',\r\n            \'src/index.ts\',\r\n            \'src/database.ts\',\r\n            \'src/controllers/setup.controller.ts\'\r\n        ],\r\n        tests: [\r\n            { pattern: \'src/**/*.spec.ts\', load: true }\r\n        ],\r\n        env: {\r\n            type: \'node\'\r\n        },\r\n\r\n        testFramework: \'jasmine\',\r\n        compilers: {\r\n            \'**/*.ts\': wallaby.compilers.typeScript(compilerOptions)\r\n        },\r\n        debug: false,\r\n        name: \'Grading Companion NodeJS\',\r\n        slowTestThreshold: 200,\r\n        lowCoverageThreshold: 90, // 90%\r\n        maxConsoleMessagesPerTest: 100,\r\n        reportConsoleErrorAsError: true,\r\n        \r\n        workers: {\r\n            restart: true\r\n        }\r\n    };\r\n};' },
  packageJSON:
   { dependencies: { 'body-parser': '1.19.0', chalk: '2.4.2', 'crypto-js': '3.1.9-1', express: '4.17.1', mysql: '2.17.1', rollbar: '2.12.3' },
     devDependencies:
      { '@types/crypto-js': '3.1.43',
        '@types/express': '4.17.1',
        '@types/jasmine': '3.4.0',
        '@types/jasminewd2': '2.0.6',
        '@types/mysql': '2.15.7',
        '@types/node': '12.7.5',
        '@types/sequelize': '4.28.4',
        jasmine: '3.4.0',
        'jasmine-core': '3.4.0',
        'jasmine-spec-reporter': '4.2.1',
        nodemon: '1.19.2',
        'ts-node': '8.3.0',
        tslint: '5.20.0',
        typescript: '3.6.3' } },
  fs: { numberOfFiles: 60 },
  debug:
   [ '2019-09-25T09:05:47.094Z project Wallaby Node version: v10.16.3\n',
     '2019-09-25T09:05:47.094Z project Wallaby config: C:\\Workspaces\\Bitbucket\\grading-companion-nodejs\\wallaby.js\n',
     '2019-09-25T09:05:47.765Z project File cache: C:\\Users\\Keith\\.vscode\\extensions\\wallabyjs.wallaby-vscode-1.0.143\\projects\\1532ed190bb385c4\n',
     '2019-09-25T09:05:47.845Z uiService Listening port 51235\n',
     '2019-09-25T09:05:47.874Z project Config file change detected, invalidating local cache\n',
     '2019-09-25T09:05:47.902Z workers Parallelism for initial run: 6, for regular run: 3\n',
     '2019-09-25T09:05:47.903Z workers Starting run worker instance #0\n',
     '2019-09-25T09:05:47.903Z workers Starting run worker instance #1\n',
     '2019-09-25T09:05:47.903Z workers Starting run worker instance #2\n',
     '2019-09-25T09:05:47.903Z workers Starting run worker instance #3\n',
     '2019-09-25T09:05:47.903Z workers Starting run worker instance #4\n',
     '2019-09-25T09:05:47.903Z workers Starting run worker instance #5\n',
     '2019-09-25T09:05:47.904Z workers Web server is listening at 57106\n',
     '2019-09-25T09:05:47.904Z project File cache requires some updates, waiting required files from IDE\n',
     '2019-09-25T09:05:48.023Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.182Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.182Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.182Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.183Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.183Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.183Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.184Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.184Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.184Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.185Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.185Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.185Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.186Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.186Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.186Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.187Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.187Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.187Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.188Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.188Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.188Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.189Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.189Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.189Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.195Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.195Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.195Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.196Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.196Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.196Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.196Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.197Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.197Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.197Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.198Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.198Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.198Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.199Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.200Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.200Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.200Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.201Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.201Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.201Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.202Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.202Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.203Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.203Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.203Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.203Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.204Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.204Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.204Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.205Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.205Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.205Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.205Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.205Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.206Z project Starting processor pool (if not yet started)\n',
     '2019-09-25T09:05:48.380Z workers Started run worker instance (delayed) #2\n',
     '2019-09-25T09:05:48.401Z workers Started run worker instance (delayed) #1\n',
     '2019-09-25T09:05:48.407Z workers Started run worker instance (delayed) #4\n',
     '2019-09-25T09:05:48.410Z workers Started run worker instance (delayed) #0\n',
     '2019-09-25T09:05:48.521Z workers Started run worker instance (delayed) #5\n',
     '2019-09-25T09:05:48.522Z workers Started run worker instance (delayed) #3\n',
     '2019-09-25T09:05:49.161Z uiService UI client connected\n',
     '2019-09-25T09:05:49.162Z uiService Outgoing message ui:handshake\n',
     '2019-09-25T09:05:49.166Z uiService Incoming message ui:tests:resultsRequested\n',
     '2019-09-25T09:05:49.167Z uiService Outgoing message ui:tests:allResultsUpdated\n',
     '2019-09-25T09:05:49.167Z uiService Incoming message ui:start\n',
     '2019-09-25T09:05:49.168Z uiService Outgoing message ui:summary\n',
     '2019-09-25T09:05:49.169Z uiService Outgoing message ui:files\n',
     '2019-09-25T09:05:49.257Z project Stopping process pool\n',
     '2019-09-25T09:05:49.258Z uiService Outgoing message ui:busy\n',
     '2019-09-25T09:05:49.260Z project Running postprocessor\n',
     '2019-09-25T09:05:49.267Z postprocessor New TypeScript language service is required\n',
     '2019-09-25T09:05:51.759Z project Postprocessor execution finished\n',
     '2019-09-25T09:05:51.759Z project Test run started; run priority: 3\n',
     '2019-09-25T09:05:51.760Z project Running all tests\n',
     '2019-09-25T09:05:51.763Z workers Starting test run, priority: 3\n',
     '2019-09-25T09:05:51.763Z workers Distributing tests between 6 workers\n',
     '2019-09-25T09:05:51.764Z workers Running tests in parallel\n',
     '2019-09-25T09:05:51.764Z nodeRunner Starting sandbox [worker #0, session #g6y2x]\n',
     '2019-09-25T09:05:51.764Z nodeRunner Starting sandbox [worker #1, session #1f1fg]\n',
     '2019-09-25T09:05:51.764Z nodeRunner Starting sandbox [worker #2, session #e2bdr]\n',
     '2019-09-25T09:05:51.764Z nodeRunner Preparing sandbox [worker #0, session #g6y2x]\n',
     '2019-09-25T09:05:51.764Z nodeRunner Preparing sandbox [worker #1, session #1f1fg]\n',
     '2019-09-25T09:05:51.764Z nodeRunner Preparing sandbox [worker #2, session #e2bdr]\n',
     '2019-09-25T09:05:51.764Z nodeRunner Prepared sandbox [worker #0, session #g6y2x]\n',
     '2019-09-25T09:05:51.764Z nodeRunner Prepared sandbox [worker #1, session #1f1fg]\n',
     '2019-09-25T09:05:51.764Z nodeRunner Prepared sandbox [worker #2, session #e2bdr]\n',
     '2019-09-25T09:05:51.764Z workers [worker #0, session #g6y2x] Running tests in sandbox\n',
     '2019-09-25T09:05:51.766Z workers [worker #1, session #1f1fg] Running tests in sandbox\n',
     '2019-09-25T09:05:51.766Z workers [worker #2, session #e2bdr] Running tests in sandbox\n',
     '2019-09-25T09:05:51.929Z workers [g6y2x] Loaded 2 test(s)\n',
     '2019-09-25T09:05:51.942Z workers [1f1fg] Loaded 5 test(s)\n',
     '2019-09-25T09:05:52.036Z workers [e2bdr] Loaded 2 test(s)\n',
     '2019-09-25T09:05:52.041Z workers [g6y2x] Test executed: should find student Tony Stark\n',
     '2019-09-25T09:05:52.047Z workers [g6y2x] Test executed: should have atleast 1 graded student\n',
     '2019-09-25T09:05:52.053Z workers [e2bdr] Test executed: findStudentGraded should find A Test user techniques known\n',
     '2019-09-25T09:05:52.070Z workers [1f1fg] Test executed: should have more than a level associated with student\n',
     '2019-09-25T09:05:52.075Z workers [1f1fg] Test executed: should have more than 1 student\n',
     '2019-09-25T09:05:52.083Z workers [g6y2x] Run 2 test(s), skipped 0 test(s)\n',
     '2019-09-25T09:05:52.084Z workers [g6y2x] Sandbox is responsive, closing it\n',
     '2019-09-25T09:05:52.085Z workers Starting run worker instance #0\n',
     '2019-09-25T09:05:52.099Z workers [e2bdr] Run 1 test(s), skipped 1 test(s)\n',
     '2019-09-25T09:05:52.101Z workers [e2bdr] Sandbox is responsive, closing it\n',
     '2019-09-25T09:05:52.101Z workers Starting run worker instance #2\n',
     '2019-09-25T09:05:52.182Z workers [1f1fg] Test executed: disabledStudent should stop the student being visible\n',
     '2019-09-25T09:05:52.185Z workers [1f1fg] Test executed: findByGuid should find A Test user\n',
     '2019-09-25T09:05:52.220Z workers [1f1fg] Test executed: updateStudentDetails should update A Test user details\n',
     '2019-09-25T09:05:52.249Z workers [1f1fg] Run 5 test(s), skipped 0 test(s)\n',
     '2019-09-25T09:05:52.250Z workers [1f1fg] Sandbox is responsive, closing it\n',
     '2019-09-25T09:05:52.251Z workers Starting run worker instance #1\n',
     '2019-09-25T09:05:52.251Z workers Merging parallel test run results\n',
     '2019-09-25T09:05:52.263Z project Test run finished\n',
     '2019-09-25T09:05:52.264Z project Processed console.log entries\n',
     '2019-09-25T09:05:52.264Z project Processed loading sequences\n',
     '2019-09-25T09:05:52.265Z project Processed executed tests\n',
     '2019-09-25T09:05:52.275Z project Processed code coverage\n',
     '2019-09-25T09:05:52.329Z project Test run result processed and sent to IDE\n',
     '2019-09-25T09:05:52.330Z uiService Outgoing message ui:summary\n',
     '2019-09-25T09:05:52.333Z uiService Outgoing message ui:tests:allResultsUpdated\n',
     '2019-09-25T09:05:52.334Z uiService Outgoing message ui:coverageChanged\n',
     '2019-09-25T09:05:52.397Z workers Started run worker instance (delayed) #0\n',
     '2019-09-25T09:05:52.412Z workers Started run worker instance (delayed) #2\n',
     '2019-09-25T09:05:52.568Z workers Started run worker instance (delayed) #1\n',
     '2019-09-25T09:07:20.314Z uiService UI client connected\n',
     '2019-09-25T09:07:20.314Z uiService Outgoing message ui:handshake\n',
     '2019-09-25T09:07:20.348Z uiService Incoming message ui:start\n',
     '2019-09-25T09:07:20.348Z uiService Outgoing message ui:summary\n',
     '2019-09-25T09:07:20.349Z uiService Outgoing message ui:files\n',
     '2019-09-25T09:07:20.350Z uiService Incoming message ui:tests:resultsRequested\n',
     '2019-09-25T09:07:20.351Z uiService Outgoing message ui:tests:allResultsUpdated\n' ] }
```

**Comment by robilars-amazon** [comment]
Having the same issue. Additionally, there is a error reported in the javascript console:
```
main.3b32e001b310d124d67d.js:1 ERROR Error: Uncaught (in promise): TypeError: Cannot read property 'path' of undefined
TypeError: Cannot read property 'path' of undefined
    at t._buildFileTree (main.3b32e001b310d124d67d.js:1)
    at t.fileTreeRootNodes (main.3b32e001b310d124d67d.js:1)
    at new t (main.3b32e001b310d124d67d.js:1)
    at t.select (main.3b32e001b310d124d67d.js:1)
    at e.activate (main.3b32e001b310d124d67d.js:1)
    at t.<anonymous> (main.3b32e001b310d124d67d.js:1)
    at main.3b32e001b310d124d67d.js:1
    at Object.next (main.3b32e001b310d124d67d.js:1)
    at a (main.3b32e001b310d124d67d.js:1)
    at t.invoke (polyfills.e5d476f3d0683921397b.js:1)
    at M (polyfills.e5d476f3d0683921397b.js:1)
    at polyfills.e5d476f3d0683921397b.js:1
    at a (main.3b32e001b310d124d67d.js:1)
    at t.invoke (polyfills.e5d476f3d0683921397b.js:1)
    at Object.onInvoke (main.3b32e001b310d124d67d.js:1)
    at t.invoke (polyfills.e5d476f3d0683921397b.js:1)
    at e.run (polyfills.e5d476f3d0683921397b.js:1)
    at polyfills.e5d476f3d0683921397b.js:1
    at t.invokeTask (polyfills.e5d476f3d0683921397b.js:1)
    at Object.onInvokeTask (main.3b32e001b310d124d67d.js:1)
to @ main.3b32e001b310d124d67d.js:1
t.handleError @ main.3b32e001b310d124d67d.js:1
next @ main.3b32e001b310d124d67d.js:1
i @ main.3b32e001b310d124d67d.js:1
e.__tryOrUnsub @ main.3b32e001b310d124d67d.js:1
e.next @ main.3b32e001b310d124d67d.js:1
e._next @ main.3b32e001b310d124d67d.js:1
e.next @ main.3b32e001b310d124d67d.js:1
e.next @ main.3b32e001b310d124d67d.js:1
e.emit @ main.3b32e001b310d124d67d.js:1
(anonymous) @ main.3b32e001b310d124d67d.js:1
t.invoke @ polyfills.e5d476f3d0683921397b.js:1
e.run @ polyfills.e5d476f3d0683921397b.js:1
t.runOutsideAngular @ main.3b32e001b310d124d67d.js:1
onHandleError @ main.3b32e001b310d124d67d.js:1
t.handleError @ polyfills.e5d476f3d0683921397b.js:1
e.runGuarded @ polyfills.e5d476f3d0683921397b.js:1
t @ polyfills.e5d476f3d0683921397b.js:1
r.microtaskDrainDone @ polyfills.e5d476f3d0683921397b.js:1
g @ polyfills.e5d476f3d0683921397b.js:1
e.invokeTask @ polyfills.e5d476f3d0683921397b.js:1
m @ polyfills.e5d476f3d0683921397b.js:1
b @ polyfills.e5d476f3d0683921397b.js:1
```

**Comment by ArtemGovorov** [comment]
Thanks for the report, the issue is fixed in [the latest core v1.0.755](https://wallabyjs.com/docs/intro/troubleshooting.html#forcing-wallabyjs-core-update).

**Comment by robilars-amazon** [comment]
Thanks for the quick fix!
